test_runner/regress/test_sharding.py

```diff
@@ -219,6 +219,11 @@ def test_sharding_split_smoke(
         placement_policy='{"Attached": 1}',
         conf=non_default_tenant_config,
     )
+    env.storage_controller.allowed_errors.extend(
+        [
+            ".*Lock on ShardSplit was held for.*",
+        ]
+    )
```

Comment:
Instead of allow listing these logs, let's bump `LOCK_TIMEOUT_ALERT_THRESHOLD` to be equal to `RECONCILER_TIMEOUT`. Most (all?) operations that grab these locks speak to the pageserver and expect it to perform some IO, so it makes sense for it to be longer at this point.

Reply:
Same for `test_sharding_split_failures` and you'll have to tweak the failpoint in `test_lock_time_tracing`.

Reply:
Makes sense, will do that

Reply:
Did you mean `SHORT_RECONCILE_TIMEOUT`(5s) or `RECONCILE_TIMEOUT`(30s)?

Reply:
`RECONCILE_TIMEOUT` (30s)